java/sage/epg/sd/SDRipper.java

```diff
@@ -2231,6 +2240,37 @@ public void run()
 
       if (Sage.DBG) System.out.println("SDEPG Downloaded " + downloadedPrograms +
         " programs for " + downloadedAirings + " airings for: " + lineup);
+
+      long currentTime;
+      if (editorialImportLimit > 0 &&
+        (currentTime = Sage.time()) >= Sage.getLong(PROP_EDITORIAL_NEXT_RUN, currentTime))
+      {
+        List<SDEditorial> editorials = SDRecommended.getUsable(SDRecommended.getRecommendations(false));
+        // Try to at least fill one screen with recommendations if possible.
+        int minRecommendations = Math.min(6, editorialImportLimit / 2);
+        if (editorials.size() < minRecommendations)
+        {
+          if (Sage.DBG) System.out.println("SDEPG Got less than " + minRecommendations +
+            " recommendations, adding intelligent recordings");
+          editorials = SDRecommended.getUsable(SDRecommended.getRecommendations(true));
+        }
+        SDRecommended.reduceByWeight(editorials, editorialImportLimit);
+        for (SDEditorial editorial : editorials)
+        {
+          Airing airing = editorial.getAiring();
+          String startTime = Sage.dfStd(airing.getStartTime());
```

Comment:
Use the date format of yyyy-MM-dd for this. It's expecting that in Wizard when it does the 'maintenance' process which basically garbage collects the DB. It'll then automatically remove editorials that are then over a week old (i.e. the date is more than a week in the past). If you use the m/d/yyyy format which Sage.dfStd applies, then it'll fail the date parsing in Wizard and never end up removing any of your added editorials. This is around line 1316 in Wizard that this happens.